Node.js/README.md

```diff
@@ -1,6 +1,6 @@
 # IBM MQ Node.js samples
 The Node.js samples are based on https://github.com/ibm-messaging/mq-mqi-nodejs/tree/master/samples
-and have been tested with Node.js v16.8.1
+and have been tested with Node.js v20.11.1
```

Comment:
Have the basicXXX set of examples been tested with this update. ie. basicget.js, basicput.js  etc.

Reply:
Yes I had run all the files in the repo when running the final regression test with the changes. All the changes made in the repo work with Node version 20.11.1

Reply:
I guess all the basicXX samples use synchronous mq operations, so they were immune to the ibmmq changes.